include/vzlist.h

```diff
@@ -117,7 +119,8 @@ struct Cveinfo {
 	unsigned long long features_mask;
 	unsigned long long features_known;
 	float vm_overcommit;
-	unsigned long nf_mask; /* netfilter */
```

Comment:
why change the comment style that you have just introduced by the previous patch?